docs/framework/unmanaged-api/diagnostics/isymunmanagedencupdate-updatesymbolstore2-method.md

```diff
@@ -44,7 +44,7 @@ HRESULT UpdateSymbolStore2(
   
 #### Parameters  
  `pIStream`  
- [in] A pointer to an <xref:IStream> that contains the line information.  
+ [in] A pointer to an `IStream` that contains the line information.  
```

Comment:
It's best for this to be a link rather than fenced; this should be either \<xref:System.Runtime.InteropServices.ComTypes.IStream> or (preferably) \[IStream interface](https://msdn.microsoft.com/library/aa380034.aspx).